src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -1031,70 +1045,84 @@ class BasicLayout extends React.PureComponent{
                 try {
-                    let bindEvt = this.id == "sDate_new_cookedDual" ? gObject.onStartDateChangedDual : this.id == "eDate_new_cookedDual" ? gObject.onEndDateChangedDual : this.id == "sDate_new_cooked" ? gObject.onStartDateChanged : this.id == "eDate_new_cooked" ? gObject.onEndDateChanged : this.id == "sDate_new_cooked2" ? gObject.onStartDate2Changed : gObject.onEndDate2Changed;
+                    let bindEvt = this.id === "sDate_new_cookedDual" ? gObject.onStartDateChangedDual : this.id === "eDate_new_cookedDual" ? gObject.onEndDateChangedDual : this.id === "sDate_new_cooked" ? gObject.onStartDateChanged : this.id === "eDate_new_cooked" ? gObject.onEndDateChanged : this.id === "sDate_new_cooked2" ? gObject.onStartDate2Changed : gObject.onEndDate2Changed;
                     console.log("i did this!");
                     $(this).datepicker({
                         changeMonth: true,
                         changeYear: true,
                         dateFormat: "yy-mm-dd",
-                        onSelect: function(dateText){ bindEvt(this.value);}
+                        onSelect: function(){ bindEvt(this.value);}
                     });
                 } catch (e) {
                     console.warn(e.message);
                 }
             });},250);
-            if(["LANDSAT5", "LANDSAT7", "LANDSAT8", "Sentinel2"].includes(this.state.selectedDataType) && this.state.wizardStep == 1){
+            if(["LANDSAT5", "LANDSAT7", "LANDSAT8", "Sentinel2"].includes(this.state.selectedDataType) && this.state.wizardStep === 1){
                 return <React.Fragment>
-                    <div className="form-group">
-                        <label htmlFor="widgetTitle">Title</label>
-                        <input type="text" name="widgetTitle" id="widgetTitle" value={this.state.WidgetTitle}
-                               className="form-control" onChange={this.onWidgetTitleChange}/>
-                    </div>
+                    {this.getTitleBlock()}
                     <label>Select the Date Range you would like</label>
                     <div className="input-group input-daterange" id="range_new_cooked">
 
                         <input type="text" className="form-control" onChange={this.onStartDateChanged} value={this.state.startDate} placeholder={"YYYY-MM-DD"} id="sDate_new_cooked"/>
                         <div className="input-group-addon">to</div>
                         <input type="text" className="form-control" onChange={this.onEndDateChanged} value={this.state.endDate} placeholder={"YYYY-MM-DD"} id="eDate_new_cooked"/>
                     </div>
-                    <div className="form-group" style={{display: this.state.selectedWidgetType == "DualImageCollection"? "none": "block"}}>
+                    <div className="form-group" style={{display: this.state.selectedWidgetType === "DualImageCollection"? "none": "block"}}>
```

Comment:
use conditional rendering instead of display none {this.state.selectedWidgetType !== "DualImageCollection" && ........}